### PR TITLE
[AUCT-892] Add LotViewed event #trivial

### PR DESCRIPTION
**Pull request by bhoggard**
Completes https://artsyproduct.atlassian.net/browse/AUCT-892 🔒 

Screen Shot 2020-03-19 at 3 13 49 PM

Paired with @yuki24 

#trivial

### CODE REVIEW

File: src/lib/Scenes/Artwork/Components/CommercialInformation.tsx [hunk 1]
```diff
@@ -24,6 +25,7 @@ interface CommercialInformationProps {
   timerState?: AuctionTimerState
   label?: string
   duration?: moment.Duration
+  tracking?: any
```
Comment (yuki24) [inline src/lib/Scenes/Artwork/Components/CommercialInformation.tsx]:
```suggestion
  tracking?: TrackingProp
```

File: src/lib/Scenes/Artwork/Components/CommercialInformation.tsx [hunk 2]
```diff
@@ -9,6 +9,7 @@ import {
 } from "lib/Components/Bidding/Components/Timer"
 import { TimeOffsetProvider } from "lib/Components/Bidding/Context/TimeOffsetProvider"
 import { StateManager as CountdownStateManager } from "lib/Components/Countdown"
+import { Schema, track } from "lib/utils/track"
```
Comment (yuki24) [inline src/lib/Scenes/Artwork/Components/CommercialInformation.tsx]:
```suggestion
import { Schema, track } from "lib/utils/track"
import { TrackingProp } from "react-tracking"
```

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>TypeScript hasn't passed, see below for full logs</td>
    </tr>
  </tbody>
</table>




  ## TSLint Issues:

  * :no_entry_sign: <a href="https://github.com/artsy/eigen/blob/auct-892-lot-viewed-event/src/lib/Scenes/Artwork/Components/CommercialInformation.tsx">CommercialInformation.tsx</a> - ordered-imports -Import sources within a group must be alphabetized.
  

### TypeScript Fails

```src/lib/Scenes/Artwork/Components/__tests__/CommercialInformation-tests.tsx(214,11): error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: Readonly<CommercialInformationProps>): CommercialInformationTimerWrapper', gave the following error.
    Property 'getTrackingData' is missing in type '{ trackEvent: jest.Mock<any, any>; }' but required in type 'TrackingProp<{}>'.
  Overload 2 of 2, '(props: CommercialInformationProps, context?: any): CommercialInformationTimerWrapper', gave the following error.
    Property 'getTrackingData' is missing in type '{ trackEvent: jest.Mock<any, any>; }' but required in type 'TrackingProp<{}>'.
src/lib/Scenes/Artwork/Components/__tests__/CommercialInformation-tests.tsx(234,11): error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: Readonly<CommercialInformationProps>): CommercialInformationTimerWrapper', gave the following error.
    Property 'getTrackingData' is missing in type '{ trackEvent: jest.Mock<any, any>; }' but required in type 'TrackingProp<{}>'.
  Overload 2 of 2, '(props: CommercialInformationProps, context?: any): CommercialInformationTimerWrapper', gave the following error.
    Property 'getTrackingData' is missing in type '{ trackEvent: jest.Mock<any, any>; }' but required in type 'TrackingProp<{}>'.
src/lib/Scenes/Artwork/Components/__tests__/CommercialInformation-tests.tsx(334,11): error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: Readonly<CommercialInformationProps>): CommercialInformationTimerWrapper', gave the following error.
    Property 'getTrackingData' is missing in type '{ trackEvent: jest.Mock<any, any>; }' but required in type 'TrackingProp<{}>'.
  Overload 2 of 2, '(props: CommercialInformationProps, context?: any): CommercialInformationTimerWrapper', gave the following error.
    Property 'getTrackingData' is missing in type '{ trackEvent: jest.Mock<any, any>; }' but required in type 'TrackingProp<{}>'.
```
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by yuki24** [comment]
The type error seems legit, perhaps caused by my suggestions. I'll look into it tomorrow morning.

File: src/lib/Scenes/Artwork/Artwork.tsx [hunk 3]
```diff
@@ -176,7 +178,7 @@ export class Artwork extends React.Component<Props, State> {
 
     sections.push({
       key: "commercialInformation",
-      element: <CommercialInformation artwork={artworkAboveTheFold} />,
+      element: <CommercialInformation artwork={artworkAboveTheFold} tracking={tracking} />,
```
Comment (yuki24) [inline src/lib/Scenes/Artwork/Artwork.tsx]: @ashfurrow I wasn't actually sure if this is the right thing to do... I'll go ahead and merge this but please do let me know if there is something better we could do.

**Pull request merged**